starlark/library.go

```diff
@@ -888,6 +889,32 @@ func print(thread *Thread, b *Builtin, args Tuple, kwargs []Tuple) (Value, error
 	return None, nil
 }
 
+// TODO: add link to spec.md
+func fail(thread *Thread, b *Builtin, args Tuple, kwargs []Tuple) (Value, error) {
+	if len(kwargs) > 0 {
+		return nil, fmt.Errorf("fail does not accept keyword arguments")
+	}
+	if len(args) == 0 {
+		return None, fmt.Errorf("none")
```

Comment:
All built-in functions should include their name in the error message. So this should be "fail: no arguments".

But UnpackPositionalArguments takes a min argument to require a minimum number of positional arguments. If you set it to 1, you don't need this check.

Reply:
My initial implementation was basically making an exact copy of fail() in Java which accepted 0 args, that's why I didn't set the min to 1.